docs/sdks/client-sdks/flutter/bandits.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Contextual Bandits
+sidebar_position: 5
+---
+
+## Usage with Contextual Multi-Armed Bandits
+
+Eppo supports contextual [multi-armed bandits](/contextual-bandits). You configure bandit flags in the Eppo application, where you can set up:
+
+- Flag keys
+- Status quo variations
+- Bandit variations
+- Targeting rules
+
+When you query the bandit in your code, you provide the available actions to the SDK.
+
+To leverage bandits using the Dart SDK, there are two additional steps over regular feature flags:
+
+1. Implement a logger that handles bandit actions
+2. Query the bandit for an action
+
+### Define a bandit assignment logger
+
+In order for the bandit to learn an optimized policy, we need to capture and log the bandit's actions.
+The Dart SDK uses a single logger class that handles both assignment and bandit action logging:
+
+```dart
+// Implement your own `BanditLogger`
+class YourBanditLogger implements BanditLogger {
+  @override
+  void logBanditEvent(BanditEvent event) {
+    // Your logging logic here
+  }
+}
+await Eppo.initialize(
+  PrecomputeArguments(subject: subject),
+  SdkOptions(
+    sdkKey: 'your-sdk-key',
+    banditLogger: YourBanditLogger(),
```

Comment:
consider also keeping `assignmentLogger: YourAssignmentLogger(),` in here as that's more realistic of what they would be doing

Reply:
I considered it but wanted to keep the focus on bandits